test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+require 'minitest'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
```

Comment:
Are all of these requires needed if we've already required minitest in the first require? As in, is autorun already imported when we require minitest?